woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -206,6 +207,54 @@ def test_accessor_getattr(sample_df):
         sample_df.ww.not_present
 
 
+def test_getitem(sample_df):
+    df = sample_df.copy()
+    df.ww.init(time_index='signup_date', index='id', name='dt_name')
+    assert list(df.columns) == list(df.ww.schema.columns)
+
+    subset = ['id', 'signup_date']
+    df_subset = df.ww[subset]
+    pd.testing.assert_frame_equal(to_pandas(df[subset]), to_pandas(df_subset))
+    assert subset == list(df_subset.ww._schema.columns)
+    assert df_subset.ww.index == 'id'
+    assert df_subset.ww.time_index == 'signup_date'
+
+    subset = ['age', 'email']
+    df_subset = df.ww[subset]
+    pd.testing.assert_frame_equal(to_pandas(df[subset]), to_pandas(df_subset))
+    assert subset == list(df_subset.ww._schema.columns)
+    assert df_subset.ww.index is None
+    assert df_subset.ww.time_index is None
+
+    subset = df.ww[[]]
+    assert len(subset.ww.columns) == 0
+    assert subset.ww.index is None
+    assert subset.ww.time_index is None
+
+    series = df.ww['age']
+    pd.testing.assert_series_equal(to_pandas(series), to_pandas(df['age']))
+    assert series.ww.logical_type == Integer
+    assert series.ww.semantic_tags == {'numeric'}
+
+    series = df.ww['id']
+    pd.testing.assert_series_equal(to_pandas(series), to_pandas(df['id']))
+    assert series.ww.logical_type == Integer
+    assert series.ww.semantic_tags == {'index', 'numeric'}
```

Comment:
@thehomebrewnerd I just realized we do `schema['semantic_tags'] = schema['semantic_tags'] - {'index'} - {'time_index'}` on iloc and loc so that the index tags don't propagate to the series. 

We should probably have that here as well when we get a series from the dataframe,  right?

Reply:
The `index` and `time_index` tags are not meaningful on a Series outside the context of a DataFrame, so I think it makes sense to remove them.

Reply:
We want to update this test so that the semantic tags in the series only contain numeric. Is that correct?

Reply:
Yep!

Reply:
✅ 80481cb